genesyscloud/resource_genesyscloud_architect_user_prompt_test.go

```diff
@@ -0,0 +1,243 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v56/platformclientv2"
+)
+
+type userPromptStruct struct {
+	resourceID  string
+	name        string
+	description string
+	resources   []*userPromptResourceStruct
+}
+
+type userPromptResourceStruct struct {
+	language   string
+	tts_string string
+	text       string
+	filename   string
+}
+
+func TestAccResourceUserPromptBasic(t *testing.T) {
+	userPromptResource1 := "test-user_prompt_1"
+	userPromptName1 := "TestUserPrompt_1"
```

Comment:
This should be concatenated with something like `strings.Replace(uuid.NewString(), "-", "", -1)` to avoid potential conflicts